src/UI/ledger/LedgerLoginContainer/index.tsx

```diff
@@ -1,41 +1,42 @@
-import React from 'react';
-import { faCircleNotch } from '@fortawesome/free-solid-svg-icons';
-import classNames from 'classnames';
+import React, { ReactNode } from 'react';
 
-import globalStyles from 'assets/sass/main.scss';
 import { useGetAccountInfo } from 'hooks/account/useGetAccountInfo';
 import { useLedgerLogin } from 'hooks/login/useLedgerLogin';
 import { ModalContainer } from 'UI/ModalContainer';
-import { PageState } from 'UI/PageState';
 
-import { OnProviderLoginType } from '../../../types';
-import { WithClassnameType } from '../../types';
+import type { OnProviderLoginType } from '../../../types';
+import type { WithClassnameType } from '../../types';
+import type { InnerLedgerComponentsClassNamesType } from './types';
+
 import { AddressTable } from './AddressTable';
 import { ConfirmAddress } from './ConfirmAddress';
+import { LedgerLoading } from './LedgerLoading';
 import { LedgerConnect } from './LedgerConnect';
 
-const ledgerWaitingText = 'Waiting for device';
+import styles from './ledgerLoginContainerStyles.scss';
 
 export interface LedgerLoginContainerPropsType
   extends OnProviderLoginType,
     WithClassnameType {
   wrapContentInsideModal?: boolean;
   onClose?: () => void;
+  customSpinnerComponent?: ReactNode;
+  customContentComponent?: ReactNode;
+  innerLedgerComponentsClassNames?: InnerLedgerComponentsClassNamesType;
 }
 
 export const LedgerLoginContainer = ({
   callbackRoute,
-  className = 'dapp-ledger-login-container',
```

Comment:
Do not remove default classNames anchors. You will introduce a breaking change. The default anchors should be defined as before.